Libraries/Components/AccessibilityInfo/NativeAccessibilityInfo.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export interface Spec extends TurboModule {
+  +isReduceMotionEnabled: (
+    resolve: (isReduceMotionEnabled: boolean) => void,
+  ) => void;
+  +isTouchExplorationEnabled: (
+    resolve: (isScreenReaderEnabled: boolean) => void,
+  ) => void;
+  +setAccessibilityFocus: (reactTag: number) => void;
+  +announceForAccessibility: (announcement: string) => void;
+
+  // RCTDeviceEventEmitter
+  +addListener: (eventName: string, handler: Function) => Object;
+  +removeListeners: (eventName: string, handler: Function) => void;
+}
+
+export default TurboModuleRegistry.getEnforcing<Spec>('AccessibilityInfo');
```

Comment:
`AccessibilityInfo` is an Android-only module. This will throw an error when required on iOS. So, we should change this up to conditionally call `TurboModuleRegistry.getEnforcing<Spec>`.

```typescript
export default (Platform.OS === 'android'
  ? TurboModuleRegistry.getEnforcing<Spec>('AccessibilityInfo')
  : null);
```